nifi-server-api/src/main/java/org/apache/nifi/NiFiServer.java

```diff
@@ -26,13 +26,13 @@
 import java.util.Set;
 
 /**
- *
+ * The main interface for declaring a NiFi-based server application
  */
 public interface NiFiServer {
 
     void start();
 
-    void initialize(NiFiProperties properties, Bundle systemBundle, Set<Bundle> bundles, ExtensionMapping extensionMapping);
+    void initialize(final NiFiProperties properties, final Bundle systemBundle, final Set<Bundle> bundles, final ExtensionMapping extensionMapping);
```

Comment:
The `final` keyword is implied, and not necessary on interface method arguments.